docs/build/configure/_index.md

```diff
@@ -264,6 +264,123 @@ The corresponding raw JSON looks like this:
 
 {{% /expand %}}
 
+## Webhooks
+
+Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud.
+For example, you can configure a webhook to send you a phone notification when your robot's battery drops below 10%.
+
+There is no webhooks tab in the Viam app config builder; use **Raw JSON** mode to configure webhooks as follows:
```

Comment:
I see what you do here but I think that sounds a bit odd/unfinished when it may sound better to simply not address that maybe there could also be a tab
```suggestion
Switch to **Raw JSON** mode to configure webhooks as follows:
```